acquire/utils.py

```diff
@@ -318,6 +328,15 @@ def check_and_set_acquire_args(
     if not args.children and args.skip_parent:
         raise ValueError("--skip-parent can only be set with --children")
 
+    if args.output_type == "zip" and args.compress_method not in ZIP_COMPRESSION_METHODS:
+        raise ValueError(
+            f"Invalid compression method for zip, allowed are: {', '.join(ZIP_COMPRESSION_METHODS.keys())}"
+        )
+    if args.output_type == "tar" and args.compress_method not in TAR_COMPRESSION_METHODS:
+        raise ValueError(
+            f"Invalid compression method for tar, allowed are: {', '.join(TAR_COMPRESSION_METHODS.keys())}"
+        )
+
```

Comment:
```suggestion
    if args.compress:
        if args.output_type == "zip" and args.compress_method not in ZIP_COMPRESSION_METHODS:
            raise ValueError(
                f"Invalid compression method for zip, allowed are: {', '.join(ZIP_COMPRESSION_METHODS.keys())}"
            )
        if args.output_type == "tar" and args.compress_method not in TAR_COMPRESSION_METHODS:
            raise ValueError(
                f"Invalid compression method for tar, allowed are: {', '.join(TAR_COMPRESSION_METHODS.keys())}"
            )

```

This should fix  the test cases I think. I don't know if you can have a zip output without compression though

Reply:
It looks like that is possible.